### ISSUE TITLE
pgcli depnds on exact version of sqlparse

### ISSUE THREAD

**Issue by mario-grgic** [dbcli/pgcli#585]
Today there was an update to sqlparse to 0.20.1 and pgcli failed to start after update since it expects exact version, not at least version.

Can the dependencies be loosened a bit so there is minimal version needed instead of exact version, otherwise pgcli is going to be broken a lot when dependencies are updated independently of pgcli.

**Comment by amjith** [comment]
Hi Mario, sorry about the inconvenience. The reason for being strict about the dependency is that we used to have a relaxed dependency on sqlparse and when 0.2.0 was release it broke everything. Since we got burned by that we've been cautious about having relaxed dependencies. Is this restricting you in some way?

**Comment by mario-grgic** [comment]
Thank you for the reply. I normally update all outdated packages periodically, and normally nothing breaks because of dependencies. pgcli is the first package I encountered that did break out of 211 packages I have installed. 

It's not a big deal I can work around this, it's just that it was a bit unexpected. 

But for pgcli it really comes down to figuring out which is more likely and desirable scenario:

1. every sqlparse update will introduce non-backward compatible changes and break pgcli
2. every sqlparse update will break pgcli, regardless if it is backward compatible or not

The outcome of 2 is that pgcli is broken every time sqlparse is updated. Outcome of 1. is that pgcli is broken only when sqlparse introduces non-backward compatible changes. One would expect this to happen way less often?
